_sass/module/_footer.scss

```diff
@@ -4,6 +4,10 @@
   background-color: $color-grey-5;
   padding-bottom: $spacing-base * 2;
 
+  a {
+    text-decoration: none;
```

Comment:
@elinaru should this be applied site-wide? Should all of our `<a>` tags have no decoration?